packages/admin/admin/src/lib/hooks/useMenu.ts

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+
+interface InitialValue {
```

Comment:
does not seem to be necessary to create interface here, do this in order file li type/*.type.ts or interface/*.interface.ts